PocketKit/Sources/Textile/Style/Style.swift

```diff
@@ -45,31 +49,35 @@ public struct Style {
     }
     
     public func with(size: FontDescriptor.Size) -> Style {
-        return Style(fontDescriptor: fontDescriptor.with(size: size), color: colorAsset, underlineStyle: underlineStyle, strike: strike)
+        return Style(fontDescriptor: fontDescriptor.with(size: size), color: colorAsset, underlineStyle: underlineStyle, strike: strike, paragraph: paragraph)
     }
 
     public func with(family: FontDescriptor.Family) -> Style {
-        return Style(fontDescriptor: fontDescriptor.with(family: family), color: colorAsset, underlineStyle: underlineStyle, strike: strike)
+        return Style(fontDescriptor: fontDescriptor.with(family: family), color: colorAsset, underlineStyle: underlineStyle, strike: strike, paragraph: paragraph)
     }
 
     public func with(weight: FontDescriptor.Weight) -> Style {
-        return Style(fontDescriptor: fontDescriptor.with(weight: weight), color: colorAsset, underlineStyle: underlineStyle, strike: strike)
+        return Style(fontDescriptor: fontDescriptor.with(weight: weight), color: colorAsset, underlineStyle: underlineStyle, strike: strike, paragraph: paragraph)
     }
 
     public func with(slant: FontDescriptor.Slant) -> Style {
-        return Style(fontDescriptor: fontDescriptor.with(slant: slant), color: colorAsset, underlineStyle: underlineStyle, strike: strike)
+        return Style(fontDescriptor: fontDescriptor.with(slant: slant), color: colorAsset, underlineStyle: underlineStyle, strike: strike, paragraph: paragraph)
     }
 
     public func with(color: ColorAsset) -> Style {
-        return Style(fontDescriptor: fontDescriptor, color: color, underlineStyle: underlineStyle, strike: strike)
+        return Style(fontDescriptor: fontDescriptor, color: colorAsset, underlineStyle: underlineStyle, strike: strike, paragraph: paragraph)
```

Comment:
I _think_ we want to be passing `color` as a parameter to the initializer here, as the argument label is `color` in the function declaration. Pretty sure `colorAsset` is the current color asset, so it wouldn't actually get updated in this case.

Reply:
Nice catch!